src/main.rs

```diff
@@ -457,30 +440,38 @@ fn main() {
             }
         };
     }
-    if distro == "true" {
+    if matches.is_present("no-distro") {
+        let _distro = "false";
+    } else {
         if let Ok(Some(dist)) = get_os_release() {
             table = add_row(table, abold, caps, borders, "DISTRO", &dist);
         }
     }
-    if kernel == "true" {
+    if matches.is_present("no-kernel") {
+        let _kernel = "false";
+    } else {
         match get_kernel_version() {
             Ok(kern) => table = add_row(table, abold, caps, borders, "KERNEL", &kern),
             Err(e) => error!("{}", e),
         }
     }
-    if window_manager == "true" {
+    if matches.is_present("no-window_manager") {
+        let _window_manager = "false";
+    } else {
         match get_window_manager() {
             Ok(wm) => table = add_row(table, abold, caps, borders, "WINDOW MANAGER", &wm),
             Err(e) => error!("{}", e),
         }
     }
-    if editor == "true" {
+    if matches.is_present("editor") {
         match get_editor() {
             Ok(ed) => table = add_row(table, abold, caps, borders, "EDITOR", &ed),
             Err(e) => error!("{}", e),
         }
     }
-    if shell == "true" {
+    if matches.is_present("no-shell") {
+        let _shell = "false";
```

Comment:
Unused variable.

Reply:
I have no idea why the unused variables are still in there. I could've swore I deleted those.

I assume that the red over them means that you already removed them? Does that mean that if I "resolve conversation" it'll automatically merge that into the pull request? If so, I can resolve all of the unused variables.

Reply:
You deleted the red line (`if shell == "true"`) and added the other `if` with `let _shell = "false";`, I made no changes to your branch. You should look at the code on your disk and push a new commit after fixing stuff.

Reply:
Oh that. I don't know how to "do nothing", so I added an unused variable there. Sounds really stupid now that I think about it.

For example:
```Rust
if matches.is_present("no-shell") {
    "do nothing"
} else {
    "do stuff"
}
```

Is there a way to actually "do nothing" that I missed?

Reply:
```rust
if !matches.is_present("no-shell") {
    "display shell"
}
```

You can either put `{ }` in the `true` branch, or use negation (`!`) and drop the `else`.

Reply:
Wow. This was all I had to do? I feel so stupid right now. I'm going to go back then and fix all of the stuff.